src/test/java/org/folio/inventory/dataimport/consumers/DataImportConsumerVerticleTest.java

```diff
@@ -163,7 +162,7 @@ public void shouldSendEventWithProcessedEventPayloadWhenProcessingCoreHandlerSuc
     // then
     String observeTopic = KafkaTopicNameHelper.formatTopicName(KAFKA_ENV_NAME, getDefaultNameSpace(), TENANT_ID, DI_COMPLETED.value());
     cluster.observeValues(ObserveKeyValues.on(observeTopic, 1)
-      .observeFor(20, TimeUnit.SECONDS)
+      .observeFor(30, TimeUnit.SECONDS)
```

Comment:
Thanks for trying to resolve this, I haven't been able to reliably run this test yet

Reply:
My only other guess here is that it's a tenant initialization problem, similar to what i encountered when updating mod-users to raml-module-builder 32.  That could account for the intermittent nature of the failure, and it could also explain why waiting longer makes the test pass more often.   I could try putting in wait code to check on the tenant initialization-except that it's not clear to me in this code where that's happening.

Reply:
I don't know either. Thank you for trying to make it more stable.